packages/cmf/src/cmfConnect.js

```diff
@@ -255,12 +256,19 @@ export default function cmfConnect({
 				super(props, context);
 				this.dispatchActionCreator = this.dispatchActionCreator.bind(this);
 				this.getOnEventProps = this.getOnEventProps.bind(this);
+				this.state = {
+					componentUuid: uuid.v4(),
+				};
```

Comment:
I would prefer this.uuid = bsonObjectid().toString().
I don t think we need to re-render it, it s very internal and defined in the constructor.